llvm/lib/Target/RISCV/RISCVInstrInfoVPseudos.td

```diff
@@ -6504,7 +6504,7 @@ defm PseudoVFWMACCBF16  : VPseudoVWMAC_VV_VF_BF_RM;
 //===----------------------------------------------------------------------===//
 // 13.8. Vector Floating-Point Square-Root Instruction
 //===----------------------------------------------------------------------===//
-let mayRaiseFPException = true, hasSideEffects = 0 in
+let mayRaiseFPException = true, hasSideEffects = 0, hasPostISelHook = 1 in
```

Comment:
It doesn't look like we have support for this today, but maybe we should add a target hook for this instead?  Having the post ISEL hook driven by tablgen (as opposed to a query on the underlying instruction), seems error prone.  Another idea might be a target specific consistency check before emission so at least we know at build time that pseudos and instructions are in sync?

Reply:
I was planning to move the hasPostISelHook=1 into the base class where has HasRoundMode is set. The flag should be set anytime HasRoundMode==1 && UsesVXRM=0.